osbuild-composer.spec

```diff
@@ -350,6 +350,12 @@ The dnf-json binary used by osbuild-composer and the workers.
 %files dnf-json
 %{_libexecdir}/osbuild-composer/dnf-json
 
+%post dnf-json
+# Fix ownership of the rpmmd cache files from previous versions where it was owned by root:root
+if [ -e /var/cache/osbuild-composer/rpmmd ]; then
```

Comment:
It may make sense to run this only on upgrades:
`if [ $1 -gt 1 ] && [ -e /var/cache/osbuild-composer/rpmmd ]; then`

Reply:
I'd rather have it always check, just in case there's some other way to get into that state (maybe removing the old package then installing the new one). The chmod checks to make sure ownership is root:root so that should prevent unnecessary changes.

Reply:
This is all fair. As I've wrote, I think that the change is acceptable as is, especially due to the multiple levels of checks (existence of the dir, chmod changing owner only if the current owner is `root:root`). Thanks.